DependencyInjection/PropelExtension.php

```diff
@@ -48,6 +49,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('propel.logging', $config['runtime']['logging']);
         $container->setParameter('propel.configuration', $config);
+        $container->setParameter('propel.usesDatabaseLoaderScript') = $this->usesDatabaseLoaderScript();
```

Comment:
@mringler I'm getting:
```
In PropelExtension.php line 52:
  Compile Error: Can't use method return value in write context
```

Reply:
`$container->setParameter('propel.usesDatabaseLoaderScript', $this->usesDatabaseLoaderScript());`